### PR TITLE
Mixed styling

### LINKED ISSUE #145: General styling
Fix the category cards
Fix the color on buttons
etc

### PR DESCRIPTION
**Pull request by mirrenil**
This PR will:

-replaces the old "free" image with one that aligns better color wise with the other cards
- fix the banner with the title on the category cards
- not render sign up in the burger menu if there is a currentuser
- change color of google button
- change label of google button on sign up page

closes #145

### CODE REVIEW

**Review by AdaHep — APPROVED** [review]
Looking good :)

**Review by Erikisak — DISMISSED** [review]
Snyggt!

File: src/Pages/SignUp.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ function SignUpPage() {
     e.preventDefault();
     try {
       googleSignIn();
-      navigate("/profile");
+      navigate("/profile/:id");
```
Comment (liinneea-a) [inline src/Pages/SignUp.tsx:40]: This will not work. If you want to navigate to profilePage it should be navigate(`/profile/${currentUser?.uid}`)

**Review by liinneea-a — CHANGES_REQUESTED** [review]
Just a tiny change!

**Pull request merged**